src/popup/popup.js

```diff
@@ -336,3 +336,15 @@ chrome.storage.sync.get('darkmode', (items) => {
     document.body.classList.add('dark');
   }
 });
+
+const buttonBackToTop = document.getElementsByClassName('button-backToTop')[0];
```

Comment:
Please move this selector to `src/popup/base.js`.

Reply:
Updated the PR, thanks for the feedback.